src/app/components/top-sellers/top-sellers.component.html

```diff
@@ -0,0 +1,22 @@
+<h1 class="titleTopSellers">Más vendidos</h1>
+<div *ngIf="productSellers && productSellers.length > 0" class="carousel-container">
+  <button *ngIf="controls" class="btn-carousel" (click)="onPrevClick()">
```

Comment:
Don't use contractions.
```suggestion
  <button *ngIf="controls" class="btn-carousel" (click)="onPreviousClick()">
```